src/main/java/frc/robot/poseestimator/PoseEstimatorConstants.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.poseestimator;
+
+
+public class PoseEstimatorConstants {
+
+	protected static final String LOG_PATH = "PoseEstimator/";
+
+	protected static final int OBSERVATION_COUNT_FOR_AVERAGED_POSE_CALIBRATION = 45;
```

Comment:
what does it mean?

Reply:
bruh, it is the amount of observations used for calculating the averaged pose